pages/course.js

```diff
@@ -128,93 +65,56 @@ class Course extends React.Component {
     }
 
-    let queues
-    if (this.props.course.queues && this.props.course.queues.length > 0) {
-      queues = this.props.course.queues.map(id => {
-        const queue = this.props.queues[id]
-        return (
-          <Queue
-            key={id}
-            onDeleteQueue={queueId => this.deleteQueue(queueId)}
-            onUpdateQueue={queueId => this.editQueue(queueId)}
-            isUserCourseStaff={this.props.isUserCourseStaff}
-            {...queue}
-          />
-        )
-      }, this)
-    } else {
-      queues = (
-        <div>
-          <ListGroupItem className="text-center text-muted pt-4 pb-4">
-            There aren&apos;t any queues right now
-          </ListGroupItem>
-        </div>
-      )
-    }
-
-    const createQueuePanel = (
-      <ListGroupItem>
-        <NewQueue
-          onCreateQueue={queue => this.createQueue(queue)}
-          onCancel={() => this.showCreateQueuePanel(false)}
-        />
-      </ListGroupItem>
-    )
-
-    const createQueueButton = (
-      <ShowForCourseStaff courseId={this.props.courseId}>
-        <ListGroupItem
-          action
-          className="text-muted"
-          onClick={() => this.showCreateQueuePanel(true)}
-        >
-          <FontAwesomeIcon icon={faPlus} className="mr-2" />
-          Create a queue
-        </ListGroupItem>
-      </ShowForCourseStaff>
-    )
-
     return (
       <Layout>
-        <Container fluid>
-          <Card className="courses-card">
-            <CardHeader className="bg-primary text-white d-flex align-items-center">
-              <CardTitle tag="h4" className="mb-0">
-                {this.props.course && this.props.course.name} Queues
-              </CardTitle>
-              <ShowForCourseStaff courseId={this.props.courseId}>
-                <Link
-                  route="courseStaff"
-                  params={{ id: this.props.courseId }}
-                  prefetch
-                  passHref
-                >
-                  <Button tag="a" color="light" size="sm" className="ml-auto">
-                    Manage Staff
-                  </Button>
-                </Link>
-              </ShowForCourseStaff>
-            </CardHeader>
-            <ListGroup flush>
-              <FlipMove
-                enterAnimation="accordionVertical"
-                leaveAnimation="accordionVertical"
-                duration={200}
+        <Container>
+          <div className="d-flex flex-wrap align-items-center mb-4">
+            <h1 className="display-4 d-inline-block mb-0 mt-3 mr-auto pr-3">
+              {this.props.course.name}
+            </h1>
+            <ShowForCourseStaff courseId={this.props.courseId}>
+              <Link
+                route="courseStaff"
+                params={{ id: this.props.courseId }}
+                prefetch
+                passHref
+              >
+                <Button tag="a" color="primary" className="mt-3">
+                  Manage Staff
```

Comment:
Would it be possible to make the manage staff button different than create queue button? Maybe either changing it to a secondary button , or adding an icon, or making it an outline button? Its a little too blue on the right side of the screen 😛

Reply:
Yeah, understandable :) UX is hard. I'll see if I can do something better.